cfme/test_requirements.py

```diff
@@ -10,61 +10,389 @@
+    assignee_id='tpapaioa',
+)
+
+satellite = pytest.mark.requirement(
+    "Configuration Management: Satellite",
+    description='Configuration Management providers, specifically Red Hat Satellite',
+    assignee_id='tpapaioa',
+)
+
+scvmm = pytest.mark.requirement(
+    'SCVMM Integration',
+    description='Support of SCVMM in MIQ/CFME',
+    assignee_id='jdupuy',
+)
+
+sdn = pytest.mark.requirement(
+    "Software Defined Networking",
+    description='Support for software defined network management on cloud providers',
+    assignee_id='mmojzis',
+)
+
+service = pytest.mark.requirement(
+    "Services",
+    description='Services, catalog items, bundles, catalogs, retirement',
+    assignee_id='nansari',
+)
+
+settings = pytest.mark.requirement(
+    "Settings",
+    description='Per-user appliance settings',
+    assignee_id='pvala',
+)
+
+smartstate = pytest.mark.requirement(
+    "Smartstate Analysis",
+    description='Smartstate analysis on all providers',
+    assignee_id='sbulage',
+)
+
+snapshot = pytest.mark.requirement(
+    "VM/Instance Snapshots",
+    description='Snapshot support and management',
+    assignee_id='apagac',
+)
+
+ssui = pytest.mark.requirement(
+    "Self-Service UI",
+    description='SSUI interface and use',
+    assignee_id='nansari',
+)
+
+stack = pytest.mark.requirement(
+    "Orchestration Stacks",
+    description='Cloud provider orchestration stack management and ordering',
+    assignee_id='nansari',
+)
+
+storage = pytest.mark.requirement(
+    "Block/Object Storage",
+    description='Cloud provider storage management for volumes and blocks',
+    assignee_id='mmojzis',
+)
+
+tag = pytest.mark.requirement(
+    "Tagging",
+    description='Tag creation, assignment, visibility',
+    assignee_id='anikifor',
+)
+
+timelines = pytest.mark.requirement(
+    "Event Timelines",
+    description='Event Timelines',
+    assignee_id='jdupuy',
+)
+
+tower = pytest.mark.requirement(
+    "Configuration Management: Tower",
+    description='Configuration Management providers, specifically Ansible Tower',
+    assignee_id='nachandr',
+)
+
+upgrade = pytest.mark.requirement(
+    "CFME Upgrade and Database Migrations",
+    description='Upgrading application and migrating databases',
+    assignee_id='jhenner',
+)
+
+v2v = pytest.mark.requirement(
+    'IMS V2V Migration',
+    description='Migration of Instance/VM from one provider to another using IMS',
+    assignee_id='sshveta',
+)
+
+vm_migrate = pytest.mark.requirement(
+    "VM/Instance Migration",
+    description='Storage/Compute migration of VM/Instances on single provider',
+    assignee_id='mnadeem',
+)
+
+vmrc = pytest.mark.requirement(
+    "VM Consoles: VMWare RC",
+    description='VMWare Remote Console support',
+    assignee_id='apagac',
+)
+
+vmware = pytest.mark.requirement(
+    "VMWare Integration",
+    description='Support of VMWare in MIQ/CFME',
+    assignee_id='kkulkarn'
+)
```

Comment:
@mshriver, are we going to skip pod as it is deprecated

Reply:
Wasn't in here already, and yeah I didn't add it because of deprecation. If you'd like I can add one in for marking existing tests?